File: src/main/java/wraith/fabricaeexnihilo/api/recipes/crucible/CrucibleRecipe.java
package wraith.fabricaeexnihilo.api.recipes.crucible;

import alexiil.mc.lib.attributes.fluid.volume.FluidVolume;
import wraith.fabricaeexnihilo.api.crafting.ItemIngredient;

public record CrucibleRecipe(ItemIngredient input, FluidVolume output) {
}

File: src/main/kotlin/exnihilofabrico/api/recipes/crucible/CrucibleRecipe.kt
package exnihilofabrico.api.recipes.crucible

import alexiil.mc.lib.attributes.fluid.volume.FluidVolume
import exnihilofabrico.api.crafting.ItemIngredient

data class CrucibleRecipe(val input: ItemIngredient, val output: FluidVolume)